src/app/transactions/transactions.module.ts

```diff
@@ -2,13 +2,31 @@ import { NgModule } from "@angular/core";
 import { ReactiveFormsModule } from "@angular/forms";
 import { CommonModule } from "@angular/common";
 import { MaterialComponentsModule } from "../material/material.module";
+import { BrowserModule } from "@angular/platform-browser";
+import { BrowserAnimationsModule } from "@angular/platform-browser/animations";
```

Comment:
you don't need to import `BrowserModule` and `BrowserAnimationsModule` here. it is imported in `app` module